dev/tools/gen_defaults/bin/gen_defaults.dart

```diff
@@ -108,4 +109,5 @@ Future<void> main(List<String> args) async {
   NavigationRailTemplate('$materialLib/navigation_rail.dart', tokens).updateFile();
   SurfaceTintTemplate('$materialLib/elevation_overlay.dart', tokens).updateFile();
   TypographyTemplate('$materialLib/typography.dart', tokens).updateFile();
+  BottomAppBarTemplate('$materialLib/bottom_app_bar.dart', tokens).updateFile();
```

Comment:
Minor nit: can we move this up under `AppBarTemplate` to be alphabetical with the rest of the template invocations?

Reply:
sure